config/settings.yml

```diff
@@ -62,6 +62,8 @@ bib_retriever:
     retriever_class_require: 'avalon/bib_retriever/sru'
 controlled_vocabulary:
   path: config/controlled_vocabulary.yml
+timeliner:
+  timeliner_url: http://localhost:3000/timeliner
```

Comment:
@phuongdh Is it okay to push settings changes?

Reply:
Hmm now that I think about it, it should probably live in `config/settings/development.yml` instead

Reply:
Split out to a different PR